dbos/workflow.go

```diff
@@ -1319,23 +1408,51 @@ func Recv[R any](ctx DBOSContext, topic string, timeout time.Duration) (R, error
 	if err != nil {
 		return *new(R), err
 	}
-	// Type check
+
+	// Handle nil message
+	if msg == nil {
+		return *new(R), nil
+	}
```

Comment:
Technically should never happen because we have a not null constraint on the message column.